### PR TITLE
Fixing bugs in bestbuy_api implementation and adding tests for best buy

### PR DESCRIPTION
**Pull request by TheFatBlue**

### CODE REVIEW

**Comment by TheFatBlue** [comment]
@TomasAndersonFang Please review the pull request :)

File: tests/test_api_walmart.py
Comment (TomasAndersonFang) [inline tests/test_api_walmart.py:1]: For Walmart API, I only return nine items per request.

**Review by TomasAndersonFang — CHANGES_REQUESTED** [review]
Please modify the wrong value in test_api_walmart.py

**Comment by TomasAndersonFang** [comment]
> Please modify the wrong value in test_api_walmart.py

@TheFatBlue

**Comment by TheFatBlue** [comment]
I've change the item number back to 9, please review it

**Review by TomasAndersonFang — COMMENTED** [review]
All solved, good jobs

**Review by TomasAndersonFang — APPROVED** [review]

**Comment by TheFatBlue** [comment]
#6 resloved.

**Pull request merged**